apps/prairielearn/src/ee/routers/lti13.ts

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express';
+import lti13InstanceRouter from '../pages/lti13Instance/lti13Instance';
+import { Lti13InstanceSchema } from '../../lib/db-types';
+import asyncHandler = require('express-async-handler');
+import logRequest = require('../../middlewares/logRequest');
+import { features } from '../../lib/features';
+
+import { loadSqlEquiv, queryOptionalRow } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(__filename);
+const router = Router({ mergeParams: true });
+
+router.use(
+  asyncHandler(async (req, res, next) => {
+    // The navbar relies on this property.
+    res.locals.urlPrefix = req.baseUrl;
+    next();
+  }),
+);
```

Comment:
This branch is a subset of another bigger LTI 1.3 development branch that has more pages (for later PRs). Do you want me to leave this there, or factor it into the future PR (I guess it could also move to their specific subpages.)

Reply:
I'd rather remove this for now, yeah.

Reply:
Done in a newer commit.